.github/CODEOWNERS

```diff
@@ -1,3 +1,5 @@
 * @cognitedata/starbase
 geospatial.py @cognitedata/geospatial
 test_geospatial.py @cognitedata/geospatial
+CHANGELOG.md @cognitedata/starbase @cognitedata/geospatial
```

Comment:
do you mind if we do those changes to CODEOWNERS @erlendvollset (and @starbase) ?
should offload you a bit